src/ui/zcl_abapgit_gui_page.clas.abap

```diff
@@ -19,10 +19,12 @@ CLASS zcl_abapgit_gui_page DEFINITION PUBLIC ABSTRACT
         page_menu  TYPE REF TO zcl_abapgit_html_toolbar,
       END OF  ty_control .
 
-    DATA ms_control TYPE ty_control .
+    CLASS-DATA gs_success_log_entry TYPE zif_abapgit_log=>ty_log_out.
```

Comment:
Mmmm, this supposes subclassing of gui_page. And currently some refactoring are done to remove the subclassing and make gui_page final. Also this is a global variable in fact which is not good. Maybe make sense to do via deferred_parts ... or maybe leave the feature without html green banner - see the main review comment